docs/src/main/paradox/spring-web.md

```diff
@@ -0,0 +1,39 @@
+# Spring Web
+
+Spring 5.0 introduced compatibility with the [Reactive Streams](https://github.com/reactive-streams/reactive-streams-jvm) 
+interfaces, that Lightbend co-lead with *Akka Streams* along with other library, such as Pivotal.
```

Comment:
perhaps ", a library interoperability standardization effort co-lead by Lightbend (with Akka Streams) along with Kaazing, Netflix, Pivotal, Red Hat, Twitter and many others" ?

Reply:
Yeah very good point, to re-stress the many-people involved and de-emphasise pivotal